default.nix

```diff
@@ -0,0 +1,78 @@
+{ coreutils,
+  curl,
+  fetchFromGitHub,
+  ffmpeg,
+  fzf,
+  gnugrep,
+  gnupatch,
+  gnused,
+  html-xml-utils,
+  lib,
+  makeWrapper,
```

Comment:
is this needed?

Reply:
yes it is used to wrap the script in nixos

Reply:
see [readme](https://github.com/BenediktBroich/lobster/tree/nixos#nixos-flake)